graphql/resolvers.go

```diff
@@ -1103,12 +1099,35 @@ func (r *queryResolver) PatchTasks(ctx context.Context, patchID string, sortBy *
 	if taskName != nil {
 		taskNameParam = *taskName
 	}
-	tasks, count, err := r.sc.FindTasksByVersion(patchID, sorter, statusesParam, variantParam, taskNameParam, sortDirParam, pageParam, limitParam, []string{})
+	var taskSorts []task.TasksSortOrder
+	if len(sorts) > 0 {
+		taskSorts = []task.TasksSortOrder{}
+		for _, singleSort := range sorts {
+			key := ""
+			switch singleSort.Key {
+			// the keys here should be the keys for the column headers of the tasks table
+			case "NAME":
```

Comment:
You should use one of these enums. https://github.com/evergreen-ci/evergreen/blob/54ee8f373872e48d07597e7c8da479bd3a813e48/graphql/models_gen.go#L526

They are defined in `schema.graphql` and types are generated for both the frontend and backend.

Reply:
fixed